lib/kdeproject_component.rb

```diff
@@ -40,6 +40,10 @@ def plasma
       @plasma ||= to_names(projects('kde/workspace'))
     end
 
+    def respond_to?(release)
```

Comment:
What is this about?

This is already a core feature of all objects...
https://ruby-doc.org/core-2.5.0/Object.html#method-i-respond_to-3F